src/module.c

```diff
@@ -4522,8 +4522,7 @@ int moduleListIteratorSeek(RedisModuleKey *key, long index, int mode) {
  * - EINVAL if key or ele is NULL.
  * - ENOTSUP if the key is of another type than list.
  * - EBADF if the key is not opened for writing.
- *
- * Note: Before Redis 7.0, `errno` was not set by this function. */
+ */
```

Comment:
Please keep this note and use "Redis OSS".

These comments in `module.c` are used for generated documentation of the module API, so in this case I think we should use the correct trademark term "Redis OSS". These notes about old versions are useful for module authors.

FYI: Here is the markdown file generated from the comments: https://github.com/valkey-io/valkey-doc/blob/main/docs/reference/modules/modules-api-ref.md. A script under utils/ is used for that.